### ISSUE TITLE
Breaking regression from 3.3.0 to 3.4.0 in milliseonds

### ISSUE THREAD

**Issue by bnb** [moment/luxon#1488]
**Describe the bug**
I use Luxon for nodevu, which is a module to get data about Node.js versions. I run nightly builds of static data for the module, and they started failing a bit ago. I recently had a user reach out saying they'd realized there was an issue with Luxon. Upon pretty basic investigation, it seems there's an issue between Luxon 3.3.0 and 3.4.0 where behavior around milliseconds seemingly changed.

**To Reproduce**
Please share a minimal code example that triggers the problem:
I wasn't able to narrow down _what_ in Luxon is triggering this, but I did create a minimal reproduction to trigger the issue with nodevu. I apologize for not being able to dig in more and make something more minimal, but I'm currently OOO and am limited on time to work on code.

```bash
git clone https://github.com/bnb/luxon-breaking-minor.git
npm install
npm start
```

That will output nodevu output that can safely be ignored _and_ the error + stack trace.

**Actual vs Expected behavior**
Milliseconds management should not change in minors.

**Desktop (please complete the following information):**

- OS: [e.g. iOS] macOS
- Browser [e.g. Chrome 84, safari 14.0] N/A
- Luxon version [e.g. 1.25.0] 3.4.0
- Your timezone [e.g. "America/New_York"] America/New_York

**Comment by diesieben07** [comment]
Thank you for the report and apologies for the breaking bug in a minor release.
I will work on a pull request with a fix.

**Comment by diesieben07** [comment]
FYI, this was caused by https://github.com/moment/luxon/pull/1467

**Comment by SudhirkumarRamani** [comment]
Hello @diesieben07,

Please suggest ETA on this issue if you could as it is breaking my app as well.

**Comment by diesieben07** [comment]
Hello @SudhirkumarRamani 

I cannot provide you an ETA unfortunately, as I am not in charge of the release schedule. However I want to stress again that this only affects _invalid_ Durations. If you are running into this issue, your app is not handling invalid Durations properly.

**Comment by SudhirkumarRamani** [comment]
Hello @diesieben07 ,

Could please suggest right person who can guide on ETA of the release. Also, app is failing in build stage itself with error as below:
<html>
<body>


ERROR in ./node_modules/luxon/src/duration.js 579:40
--
214 | Module parse failed: Unexpected token (579:40)
215 | You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
216 | \|    */
217 | \|   toMillis() {
218 | >     let sum = this.values.milliseconds ?? 0;
219 | \|     for (let unit of reverseUnits.slice(1)) {
220 | \|       if (this.values?.[unit]) {
221 | @ ./node_modules/luxon/src/luxon.js 2:0-37 14:0-26:2
222 | @ ./node_modules/cron-parser/lib/date.js
223 | @ ./node_modules/cron-parser/lib/expression.js
224 | @ ./node_modules/cron-parser/lib/parser.js


</body>
</html>

**Comment by diesieben07** [comment]
@SudhirkumarRamani Your issue is different. Your issue is caused by an outdated version of Webpack, which does not support "nullish coalescing" or "optional chaning". See this Webpack issue: https://github.com/webpack/webpack/issues/10227

The primary maintainer for this package is @icambron, who I have already tagged in the pull request which will fix this issue. However please note again that your issue is unrelated.

**Comment by icambron** [comment]
The fix has been released in 3.4.1
